### PR TITLE
Fixes windoors of the robotics lab on nss kerberos and nss farragus are not electrochromic

### LINKED ISSUE #23686: The windoors of the robotics lab on NSS Kerberosand NSS Farragus are not electrochromic.
### Exploit Reports

- [X] I confirm this issue is not an exploit. (Required)

### BYOND Version

515.1620

### Issue Description

The Robotics surgery areas of Kerberos and Farragus both are surrounded by electrochromic windows. Both are entered by a single windoor. The windoors are not electrochromic even though the windows on either side of them are.

### What did you expect to happen?

For the electrochromic panel in the windoors to be enabled by default.

### What happened instead?

The electrochromic panel in the windoors are disabled by default.

### Why is this bad/What are the consequences?

The non-electrochromic windoors allow a hole in the privacy shield that the electrochromic windows are supposed to provide. 

### Steps to reproduce the issue.

Exist in the robotics lab of Kerberos/Farraugs.
Press the window tint button.
Sob as the door remains transparent.

### When did the problem start happening?

Since forever.

### Extra information

None.

### Relevant log output/runtime error

_No response_

### PR DESCRIPTION
**Pull request by Fluffyzack21**
## What Does This PR Do
 makes it so you have privacy again while doing surgery

 fixes #23686 

## Why It's Good For The Game
 adds privacy while doing surgery instead of being open to the public

## Images of changes
 

https://github.com/ParadiseSS13/Paradise/assets/98861947/2404a840-8e6c-4b75-aa93-07fae1c65f20

https://github.com/ParadiseSS13/Paradise/assets/98861947/ed0d15b2-ca7c-4600-8d8d-6d66ab16d4f5

## Testing
 spawned as a roboticst and tested if they worked and they did

## Changelog
:cl:
tweak: you can now toggle on nss kerberos and nss farragus your electrochromic switch and you will now be more in private room
/:cl:

### CODE REVIEW

File: _maps/map_files/Delta/delta.dmm [hunk 1]
```diff
@@ -78721,12 +78721,10 @@
 /area/station/public/fitness)
 "hQz" = (
 /obj/machinery/door/window/classic/normal{
-	name = "Robotics Desk";
-	dir = 8
-	},
-/obj/effect/mapping_helpers/airlock/windoor/access/any/science/robotics{
```
Comment (warriorstar-orion) [inline _maps/map_files/Delta/delta.dmm:78727]: It looks like the access helper for the indoors was removed.

File: _maps/map_files/cerestation/cerestation.dmm [hunk 2]
```diff
@@ -95725,13 +95725,14 @@
 /turf/simulated/floor/carpet/royalblack,
 /area/station/command/office/captain)
 "xNF" = (
-/obj/machinery/door/window{
-	dir = 8;
-	name = "Robotics Surgery"
-	},
 /obj/machinery/atmospherics/unary/vent_pump/on{
 	dir = 1
 	},
```
Comment (warriorstar-orion) [inline _maps/map_files/cerestation/cerestation.dmm:95730]: It also looks like there was never an access helper on this windoor in the first place. Could you add that as part of this PR?

**Review by warriorstar-orion — CHANGES_REQUESTED** [review]

File: _maps/map_files/Delta/delta.dmm (hunk 1 above)
Comment (Fluffyzack21) [reply _maps/map_files/Delta/delta.dmm:78727]: Just fixed it 
Screenshot 2024-01-06 164054

File: _maps/map_files/cerestation/cerestation.dmm (hunk 2 above)
Comment (Fluffyzack21) [reply _maps/map_files/cerestation/cerestation.dmm:95730]: Fixed as well 
Screenshot 2024-01-06 163929

Comment (Fluffyzack21) [reply _maps/map_files/cerestation/cerestation.dmm:95730]: with both robotics airlock helpers

File: _maps/map_files/Delta/delta.dmm (hunk 1 above)
Comment (Fluffyzack21) [reply _maps/map_files/Delta/delta.dmm:78727]: with robotics airlock helper

**Review by 1080pCat — APPROVED** [review]
Looks good

**Review by warriorstar-orion — APPROVED** [review]
Thanks for making the changes, LGTM!

**Comment by Fluffyzack21** [comment]
That is a fuck up, what the fuck

File: .vscode/settings.json [hunk 3]
```diff
@@ -22,5 +22,6 @@
 	},
 	"[scss]": {
 		"editor.rulers": [80]
-	}
+	},
+	"GitHooks.hooksDirectory": "c:\\Users\\User1\\OneDrive\\Documents\\GitHub\\Paradise\\.git\\hooks"
```
Comment (S34NW) [inline .vscode/settings.json:26]: Remove this change please, then this is good to go.

Comment (S34NW) [inline .vscode/settings.json:26]:
```suggestion
	}
```

Comment (Fluffyzack21) [reply .vscode/settings.json]: removed

Comment (Fluffyzack21) [reply .vscode/settings.json]: and removed

Comment (S34NW) [reply .vscode/settings.json]: You deleted the whole file...

Comment (Fluffyzack21) [reply .vscode/settings.json]: Re-added

Comment (S34NW) [reply .vscode/settings.json]: Nope

Comment (Fluffyzack21) [reply .vscode/settings.json:26]: okay I fixed it again

Comment (S34NW) [reply .vscode/settings.json]: now it's fixed

**Review by S34NW — APPROVED** [review]

**Review by warriorstar-orion — APPROVED** [review]
LGTM

**Review by S34NW — APPROVED** [review]

**Pull request merged**